corehq/apps/smsbillables/models.py

```diff
@@ -384,7 +393,11 @@ def _get_gateway_fee(cls, backend_api_id, backend_instance,
             backend_instance) or toggles.ENABLE_INCLUDE_SMS_GATEWAY_CHARGING.enabled(domain)
 
         if is_gateway_billable:
-            if backend_api_id in [SQLTwilioBackend.get_api_id(), InfobipBackend.get_api_id()]:
+            if backend_api_id in [
+                SQLTwilioBackend.get_api_id(),
+                InfobipBackend.get_api_id(),
+                PinpointBackend.get_api_id()
+            ]:
```

Comment:
Same here. This should delegate to the backend rather than checking a hard-coded list of backend IDs.